pages/settings/subscriptions/index.js

```diff
@@ -1,30 +1,72 @@
-import { useMemo } from 'react'
+import { useMemo, useState } from 'react'
 import { getGetServerSideProps } from '@/api/ssrApollo'
 import Layout from '@/components/layout'
+import { Select } from '@/components/form'
 import UserList from '@/components/user-list'
+import TerritoryList from '@/components/territory-list'
 import { MY_SUBSCRIBED_USERS } from '@/fragments/users'
+import { MY_SUBSCRIBED_SUBS } from '@/fragments/subs'
 import { SettingsHeader } from '../index'
 import { SubscribeUserContextProvider } from '@/components/subscribeUser'
 
-export const getServerSideProps = getGetServerSideProps({ query: MY_SUBSCRIBED_USERS, authRequired: true })
+export const getServerSideProps = async (ctx) => {
+  const [userRes, subsRes] = await Promise.all([
+    getGetServerSideProps({ query: MY_SUBSCRIBED_USERS, authRequired: true })(ctx),
+    getGetServerSideProps({ query: MY_SUBSCRIBED_SUBS, authRequired: true })(ctx)
+  ])
+  return {
+    props: {
+      ssrUsers: userRes.props.ssrData,
+      ssrSubs: subsRes.props.ssrData
+    }
+  }
+}
```

Comment:
I'm surprised this worked.

The move here is to make a compound graphql query like `USER_WITH_ITEMS` or make this into different `.js` files to call each query only when it is needed like we do for `/top` path.